core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/MifosTopBar.kt

```diff
@@ -28,18 +28,24 @@ fun MifosTopBar(
     backPress: () -> Unit,
     modifier: Modifier = Modifier,
     actions: @Composable RowScope.() -> Unit = {},
+    titleColor: Color? = null,
+    iconTint: Color? = null,
 ) {
     CenterAlignedTopAppBar(
         title = {
-            Text(
-                text = stringResource(id = topBarTitle),
-                style = MaterialTheme.typography.titleMedium,
-            )
+            if (titleColor != null) {
+                Text(
+                    text = stringResource(id = topBarTitle),
+                    style = MaterialTheme.typography.titleMedium,
+                    color = titleColor,
+                )
+            }
```

Comment:
missing logic for titleColor == null

do

```
  Text(
                    text = stringResource(id = topBarTitle),
                    style = MaterialTheme.typography.titleMedium,
                    color = titleColor ?: ColorHere,
                )
```

Reply:
From mifos scaffold I am passing default color so there is chance of getting null value